va/va.h

```diff
@@ -1028,6 +1028,13 @@ typedef enum {
      * The value returned uses the VAConfigAttribValEncPerBlockControl type.
      */
     VAConfigAttribEncPerBlockControl    = 55,
+    /** \brief dynamic resolution change support. Read-only.
+     *
+     * This attributes describes whether backend driver support dynamic resolution change
```

Comment:
Correct the grammar: This attribute(-s) describes whether the backend driver supports(+s)...indicate whether needing to reset context.